tests/worktree_spec.lua

```diff
@@ -431,15 +432,188 @@ describe('git-worktree', function()
             1000
             )
 
-            git_worktree:reset()
-
             -- Check to make sure directory was switched
             assert.are.same(vim.loop.cwd(), path)
 
         end))
 
-        pending("in a featB worktree with file A open, switch to featC and switch to file A in other worktree")
-        pending("in a featB worktree with file B open, switch to featC and switch to worktree root in other worktree")
+        local get_current_file = function()
```

Comment:
could you get me the command you use to run all the tests?  I could put that in a workflow to add to the commit hook on github.

Reply:
was planning to add a git action, i just use "make test" in the git folder